### ISSUE TITLE
training on a new dataset

### ISSUE THREAD

**Issue by blackunicorn15** [weiliu89/caffe#35]
I get this error everytime i run ssd_pascal.py . Any idea how to fix this ?

I0712 13:04:50.931237 23455 net.cpp:91] Creating Layer conv1_1
I0712 13:04:50.931251 23455 net.cpp:425] conv1_1 <- data_data_0_split_0
I0712 13:04:50.931257 23455 net.cpp:399] conv1_1 -> conv1_1
F0712 13:04:50.938412 23468 data_transformer.cpp:196] Check failed: datum_height == height (600 vs. 300) 
*** Check failure stack trace: ***
    @     0x7fccd7752daa  (unknown)
    @     0x7fccd7752ce4  (unknown)
    @     0x7fccd77526e6  (unknown)
    @     0x7fccd7755687  (unknown)
    @     0x7fccd7fe7ec0  caffe::DataTransformer<>::Transform()
    @     0x7fccd7fec0db  caffe::DataTransformer<>::Transform()
    @     0x7fccd7fec198  caffe::DataTransformer<>::Transform()
    @     0x7fccd7fec24e  caffe::DataTransformer<>::Transform()
    @     0x7fccd7f25cdf  caffe::AnnotatedDataLayer<>::load_batch()
    @     0x7fccd7efd4a9  caffe::BasePrefetchingDataLayer<>::InternalThreadEntry()
    @     0x7fccd7fdf530  caffe::InternalThread::entry()
    @     0x7fcccca565aa  thread_proxy
    @     0x7fccc85ba184  start_thread
    @     0x7fccd638737d  (unknown)
    @              (nil)  (unknown)
Aborted (core dumped)

**Comment by weiliu89** [comment]
You should store the image in the encoded format. Have you set [this](https://github.com/weiliu89/caffe/blob/ssd/data/VOC0712/create_data.sh#L17) correctly? For example, you should set the `encode_type` to your image file extension (e.g. png or JPEG or others).

**Comment by blackunicorn15** [comment]
Your suggestion was spot on. Thanks for that :)
The training procedure went well but I am facing a weird issue during testing.
My new dataset has 10 classes (including the background class). I created a new labelmap file as required by the setup. 
While testing, I found that only top 8 classes are detected across the dataset and the last two classes specified in the labelmap are not detected in any image in the dataset.
Any suggestions as to what might be the issue ?
